.github/workflows/release.yml

```diff
@@ -0,0 +1,129 @@
+
+name: Build release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to build'
+        required: false
+        default: ''
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: "Check if user is in group"
+      run: |
+          MEMBER=$( curl -s \
+                 --request GET \
+                 --header 'authorization: Bearer ${{ secrets.GITTOKEN }}' \
+                 --header 'content-type: application/json' \
+                 --url https://api.github.com/orgs/matomo-org/teams/release-team/members \
+                 | jq '.[] | select(.login == "${{ github.actor }}")' )
+          
+          if [[ -z "$MEMBER" ]]
+          then
+           echo "Action was not triggered by a member with release permission"
+           exit 1;
+          fi
+      shell: bash
+    - uses: actions/checkout@v2
+      with:
+        lfs: false
+    - name: Import GPG key
+      id: import_gpg
+      uses: crazy-max/ghaction-import-gpg@v3
+      with:
+        gpg-private-key: ${{ secrets.GPG_CERTIFICATE }}
+        passphrase: ${{ secrets.GPG_CERTIFICATE_PASS }}
+    - name: Add SSH key
+      env:
+        SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+      run: |
+          mkdir -p /home/runner/.ssh
+          ssh-keyscan matomo.org >> /home/runner/.ssh/known_hosts
+          # DOKKU_SSH_KEY is the name of the repository secret
+          echo "${{ secrets.SSH_KEY }}" > /home/runner/.ssh/github_actions
+          chmod 600 /home/runner/.ssh/github_actions
+          ssh-agent -a $SSH_AUTH_SOCK > /dev/null   
+          ssh-add /home/runner/.ssh/github_actions
+    - name: Check preconditions, create tag, build and publish release
+      id: tag
+      env:
+        SSH_AUTH_SOCK: /tmp/ssh_agent.sock
+      run: |
```

Comment:
I think this script should really have a set -e as it should not rather fail than continue after a failing step.

Reply:
This is done internally. See https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell